### ISSUE TITLE
Error C2065 while compiling in python

### ISSUE THREAD

**Issue by I-love-study** [nu-book/zxing-cpp#277]
Hello, I'm glad to see there's a convient way to scan barcode in Python.
But, when I want to build zxing-cpp in Python 3.10 on Windows with msvc(which is not have a wheel version in PyPI).
It's failed
```
zxing.cpp(94,7): error C2065: “ssize_t”: The identifier is undeclared
...
```
Is there anyway to build it success?
(Sorry for my poor English because I'm not a native speaker)

**Comment by axxel** [comment]
It sounds like you are not using the current HEAD revision. This problem was fixed 3 months ago: https://github.com/nu-book/zxing-cpp/commit/add45aac78cc188650f9f285157b6506868a178e. That was after release 1.2.0.

**Comment by I-love-study** [comment]
Oh, after download the latest zxing.cpp, it's working.
I think it's nice to publish a `1.2.1` version to PyPI, isn't it?

**Comment by axxel** [comment]
I build a pre-release 1.3.0 and published it on https://test.pypi.org/project/zxing-cpp/1.3.0/. In contains python 3.10 wheels for Windows. I'd be thankful for every feedback.

**Comment by axxel** [comment]
fixed in 1.3.0 release
